sdccc/src/main/java/com/draeger/medical/sdccc/configuration/TestSuiteConfig.java

```diff
@@ -25,6 +25,7 @@ public final class TestSuiteConfig {
     public static final String TEST_EXECUTION_LOGGING = SDCCC + "TestExecutionLogging";
     public static final String SUMMARIZE_MESSAGE_ENCODING_ERRORS = SDCCC + "SummarizeMessageEncodingErrors";
     public static final String ENABLE_MESSAGE_ENCODING_CHECK = SDCCC + "EnableMessageEncodingCheck";
+    public static final String CONFIGURATION_MODULE = "ConfigurationModule";
```

Comment:
I think it should not be part of this config class/file, especially not grouped with the general config options under "General configuration".

Reply:
done, moved it to Constants.java